lib/query/main.js

```diff
@@ -12,23 +12,26 @@ const dispatchQuery = (conn, config, params) => {
 
   const body = buildQuery(config.query, params);
 
-  return fetch(conn.uri(config.database, 'query'), {
+  return fetch(conn.uri(config.database, config.resource), {
     method: 'POST',
     body: qs.stringify(body),
     headers,
   }).then(httpBody);
 };
 
-const execute = (conn, database, query, params) =>
-  dispatchQuery(
+const execute = (conn, database, query, params) => {
+  const type = queryType(query);
+  return dispatchQuery(
     conn,
     {
       database,
       query,
       accept: mimeType(query),
+      resource: type === 'update' ? 'update' : 'query',
```

Comment:
I don't understand the question. The type is what it is, I'm just computing what endpoint to hit based on the `type`. 

Can you clarify the question?